configure.ac

```diff
@@ -224,6 +230,33 @@ if test "$enable_debug" = "yes"; then
     fi
 fi
 
+if test "$use_sanitizers" != ""; then
+  dnl First check if the compiler accepts flags. If an incompatible pair like
+  dnl -fsanitize=address,thread is used here, this check will fail. This will also
+  dnl fail if a bad argument is passed, e.g. -fsanitize=undfeined
```

Comment:
"undfeined" is misspelled

Reply:
"This will also fail if a bad argument is passed, e.g. -fsanitize=undfeined"

Reply:
;)